.circleci/config.yml

```diff
@@ -175,45 +183,49 @@ jobs:
       - store_test_results:
           path: _reports
 
-  e2e_testnet_leanhelix:
+  e2e_testnet:
     docker:
       - image: circleci/golang:1.12.9
+    environment:
+      BOOTSTRAP_URL: "https://boyar-testnet-bootstrap.s3-us-west-2.amazonaws.com/boyar/config.json"
     steps:
       - checkout
+      - attach_workspace:
+          at: workspace
       - run: ./.circleci/retry.sh 5 go get -u github.com/orbs-network/go-junit-report
-      - run: sleep 300
-      - run: ./.circleci/e2e-testnet.sh 2013 2011
+      - run: ./.circleci/e2e-testnet.sh
       - store_artifacts:
           path: _out
           destination: test-output
       - store_test_results:
           path: _reports
+      - store_artifacts:
+          path: _out
 
-  deploy_testnet:
+  success:
```

Comment:
I still don't understand why we need this job

Reply:
If you look closely at our diagram of the new CI all the requires jobs to decide whether or not we should carry on with releasing a ga and onwards are bound to a all of these requires jobs passing (the ones in the `require:` section of the success job)  without this , flakiness for example might fail or races and we would still deploy master to the audit node which according to the diagram if I remember correctly is not what should happen.

![mermaid-diagram-20191201170102](https://user-images.githubusercontent.com/5961787/71075148-42518c00-218c-11ea-8db8-3349638e8658.png)

Notice where it says "master"

Reply:
@electricmonk The `success` job is the same as the `master` step in the diagram - it is a gatekeeper for subsequent jobs.
There's no other place to assert all mandatory prerequisite jobs have passed.

Reply:
Flakiness and Races are optional.
the Release GA job should explicitly declare what it depends on. IMO these are:
- tests
- e2e
- testnet e2e
- marvin
- anything else?